charts/seed-controlplane/charts/etcd/templates/allow-from-apiserver-network-policy.yaml

```diff
@@ -6,6 +6,10 @@ metadata:
       Allows Ingress to etcd pods from the Shoot's Kubernetes API Server.
   name: allow-etcd
   namespace: {{ .Release.Namespace }}
+  labels:
+    garden.sapcloud.io/role: controlplane
```

Comment:
Should we have `gardener.cloud` instead?

Reply:
This requires more migration effort, I guess.

Reply:
It is just label changes, right? Would that be significant migration effort?

Reply:
Hm, maybe not, but I would not bundle this together with the etcd(-druid) changes now. There is #1649 which tackles this "migration".

Reply:
This cannot be bundled with etcd-druid changes. THis has to go in first and released else druid will not be able to adopt existing etcd related resources like bootstrap configmap.

Reply:
> This cannot be bundled with etcd-druid changes. THis has to go in first and released else druid will not be able to adopt existing etcd related resources like bootstrap configmap.

To be clear, in which PR it does is unimportant but the label changes have to go in and a release should happen before the etcd-druid change goes in.

Reply:
Understood. We can make this happen. :)